cmd/slinky/main.go

```diff
@@ -90,11 +91,18 @@ func init() {
 		"info",
 		"Log level (debug, info, warn, error, dpanic, panic, fatal).",
 	)
+	rootCmd.Flags().StringVarP(
+		&fileLogLevel,
+		"file-log-level",
+		"",
+		"info",
```

Comment:
can move this to debug once we purge and bubble up some of the logs.

Reply:
make linear task for this?